lib/vsc/utils/missing.py

```diff
@@ -57,7 +58,12 @@
 
 def is_string(item):
```

Comment:
should be moved to th epy23 module magic

Reply:
done